docker/docker-compose.yml

```diff
@@ -4,13 +4,13 @@ version: "3"
 services:
   relay_alice:
     container_name: alice
-    image: "parity/polkadot:v0.9.42"
+    image: "parity/polkadot:v0.9.43"
     platform: linux/x86_64
     hostname: relay_alice
     ports:
       - "30335:30335" # p2p port
-      - "9935:9933" # rpc port
-      - "9946:9944" # ws port
+      - "9946:9944" # ws and rpc port
+      - "9615:9615" # telemetry (prometheus)
```

Comment:
On relay_alice and Relay_bob we can skip telemetry as they are not rococo or mainnet. (Either way we'd need to fix that alice and bob are binding to the same local port)